app/models/user.rb

```diff
@@ -52,7 +53,7 @@ class User < ApplicationRecord
   scope :search_by_name_or_phone, ->(term) { search_by_name(term).union(search_by_phone(term)) }
 
   validates :full_name, presence: true
-  validates :role, presence: true, if: -> { email_authentication.present? }
+  validates :designation, presence: true, if: -> { email_authentication.present? }
```

Comment:
Ah, we rename this to `designation` to avoid clashing with the new "roles"? Does this need a migration?